sys-apps/yarn/yarn-0.17.10.ebuild

```diff
@@ -0,0 +1,27 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+DESCRIPTION="Fast, reliable, and secure node dependency management"
+HOMEPAGE="https://yarnpkg.com"
+SRC_URI="https://github.com/yarnpkg/yarn/releases/download/v${PV}/yarn-v${PV}.tar.gz"
+
+LICENSE="BSD-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+RDEPEND="net-libs/nodejs"
+DEPEND="${RDEPEND}"
+
+S="${WORKDIR}/dist"
+
+src_install() {
+	local install_dir="/usr/$(get_libdir)/node_modules/${PN}"
+	insinto "${install_dir}"
+	doins -r .
+	dosym "${install_dir}/bin/yarn.js" "/usr/bin/yarn"
```

Comment:
I'm sorry for failing to notice this earlier. Please use relative symlink paths whenever possible